tools/gn

```diff
@@ -147,15 +147,21 @@ def to_gn_args(args):
     gn_args['skia_use_wuffs'] = True
     gn_args['skia_use_expat'] = args.target_os == 'android'
     gn_args['skia_use_fontconfig'] = args.enable_fontconfig
-    gn_args['flutter_use_fontconfig'] = args.enable_fontconfig
-    gn_args['flutter_enable_skshaper'] = args.enable_skshaper
+    if args.target_os != 'wasm':
+      gn_args['flutter_use_fontconfig'] = args.enable_fontconfig
+      gn_args['flutter_enable_skshaper'] = args.enable_skshaper
     if args.enable_skshaper:
       gn_args['skia_use_icu'] = True
-      gn_args['flutter_always_use_skshaper'] = args.always_use_skshaper
+      if args.target_os != 'wasm':
+        gn_args['flutter_always_use_skshaper'] = args.always_use_skshaper
+      else:
+        gn_args['skia_use_harfbuzz'] = True
+        gn_args['icu_use_data_file'] = False
     if args.target_os == 'winuwp':
       gn_args['skia_enable_winuwp'] = True
     gn_args['is_official_build'] = True    # Disable Skia test utilities.
-    gn_args['dart_component_kind'] = 'static_library' # Always link Dart in statically.
+    if args.target_os != 'wasm':
+      gn_args['dart_component_kind'] = 'static_library' # Always link Dart in statically.
```

Comment:
Is the idea that this arg is unused by wasm an GN/Ninja complain?

Reply:
Yes. If an arg is unused then `gn` will fail.